### DISCUSSION TITLE
Goodness-of-fit tests

### DISCUSSION THREAD

**Discussion by TristanFauvel** [sebp/scikit-survival#318] [General]
Hello,
I would like to inquire about the relevance of submitting a pull request to add goodness-of-fit tests for survival analysis (Grønnesby & Borgan test, and translations of the Hosmer-Lemeshow test by d’Agostino & Nam and Cook & Ridler). I have implemented these tests on top of sksurv for a personal project.  Thanks.

**Comment by sebp** [comment]
Contributions are always welcome! Please open a PR and I'd be happy to review the changes.
